src/en/flamecomics/src/eu/kanade/tachiyomi/extension/en/flamecomics/FlameComics.kt

```diff
@@ -51,21 +51,18 @@ class FlameComics : HttpSource() {
         addPathSegment(buildId)
     }
 
-    private fun imageApiUrlBuilder(dataUrl: String) = baseUrl.toHttpUrl().newBuilder().apply {
-        addPathSegment("_next")
-        addPathSegment("image")
-    }.build().toString() + "?url=$dataUrl"
+    private fun imageApiUrlBuilder() = cdn.toHttpUrl().newBuilder().apply {
+        addPathSegment("uploads")
+        addPathSegment("images")
+        addPathSegment("series")
+    }
```

Comment:
```suggestion
    private fun imageApiUrlBuilder() = "$cdn/uploads/images/series".toHttpUrl().newBuilder()
```

Improves grep-ability, and works since all segments are static.